content/tutorials/manage-the-rippled-server/configuration/run-rippled-as-a-validator.md

```diff
@@ -1,74 +1,191 @@
 # Run rippled as a Validator
 
-Running a `rippled` validator that participates in the Consensus process is simple:
+A `rippled` server running in validator mode does everything a stock server does: ***TODO: I see stock servers referred to as "tracking servers" too - should we always call them one or the other? "Tracking servers" is a little more descriptive and it used in the Consensus article: https://developers.ripple.com/consensus.html. Or would it be confusing because "tracking" is also a server state: 	The server is in agreement with the network?***
```

Comment:
Lets avoid tracking for now (I also like it more), this is something we need to align on with marketing.